node_modules/oae-meetups/lib/restmodel.js

```diff
@@ -0,0 +1,14 @@
+/*!
```

Comment:
Or being removed. Is it even required for this implementation?. The files used where the ones we received as a template one year ago.